[DOCS-3964] Add more details to troubleshooting section in Eventbridge guide (#15358)
DataDog/documentation: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md

1. Follow the steps in the [Amazon Create an API destination docs][5] to add Datadog as an API destination.
    - Use API key authorization, with `DD-API-KEY` as your key name and your [Datadog API key][3] as the value.
    - For your destination endpoint, use `https://http-intake.logs.datadoghq.com/v1/input` for logs or `https://api.datadoghq.com/api/v1/events` for events, and set `POST` as the HTTP method. For more information about the differences between logs and events, see the [logs section][6], and the [events section][7] of the [Categories of Data docs page][8].

>     - If you are utilizing the events endpoint, you will need to include a `title` and `text` as `body.field` parameters in the API Destination connection. These are required values to `POST` to the events endpoint, see [Post an event API documentation][9]

Reviewer: ```suggestion
    - If you are utilizing the events endpoint, you need to include a `title` and `text` as `body.field` parameters in the API Destination connection. These are required values to `POST` to the events endpoint. For more information, see the [Post an event documentation][9].
```

> 2. Once you've set up the destination, you can now follow the Amazon instructions to [create an EventBridge rule][10], where you set Datadog as your destination.

Reviewer: ```suggestion
2. Once you have set up the destination, see the Amazon documentation to [create an EventBridge rule][10], where you set Datadog as your destination.
```

> 3. Once you've set up the rule with Datadog as the destination, trigger an event by posting an event to EventBridge. For more information about pushing events to EventBridge from Datadog, see the [EventBridge integration docs][1]. For example, to trigger a test event by [uploading the objects to an S3 bucket][11] in your account, use this AWS CloudShell command:

Reviewer: ```suggestion
3. Once you have set up the rule with Datadog as the destination, trigger an event by posting an event to EventBridge. For more information about pushing events to EventBridge from Datadog, see the [EventBridge integration documentation][1]. For example, to trigger a test event by [uploading the objects to an S3 bucket][11] in your account, use this AWS CloudShell command:
```

    ```bash
    echo "test" > testfile.txt
    aws s3 cp testfile.txt s3://YOUR_BUCKET_NAME
    ```

> 4. Once events and logs are sending, after about five minutes, the data is available in the Datadog [logs console][12] or [events explorer][13], depending on which endpoint you are sending them to.

Reviewer: ```suggestion
4. Once events and logs are sending, after about five minutes, the data is available in the Datadog [Logs Console][12] or [Events Explorer][13], depending on which endpoint you are sending them to.
```

## Troubleshooting

> You can set up an Amazon SQS queue to see more details regarding the payloads sent to Datadog and view the response of the API endpoints. 

Reviewer: ```suggestion
To see more details about the payloads sent to Datadog and to view the response of the API endpoints, set up an Amazon SQS queue:  
```

> 1. Create a new queue in [Amazon SQS][14].

Reviewer: ```suggestion
1. Create a queue in [Amazon SQS][14].
```

2. Go to the [EventBridge rule][15] that you created in the [Configuration](#configuration) section.

> 3. Select the **Targets** tab, and then click **Edit**.

Reviewer: ```suggestion
3. Select the **Targets** tab and click **Edit**.
```
